docs/python_client/examples.md

```diff
@@ -0,0 +1,158 @@
+# Examples
+
+## monoDrive Python Examples
+
+The monoDrive Python Client comes with several examples for connecting to and
+using the the monoDrive Simulator and Scenario Editor. The examples in the
+`examples` directory have many common elements that are discussed here.
+
+### Connecting to the Simulator
+
+To connect to the monoDrive Simulator, construct a `Simulator` object:
+
+```python
+simulator = Simulator.from_file(
+    "examples/configurations/simulator_closed_loop.json",
+    trajectory="examples/trajectories/Closed_Loop.json",
+    sensors="examples/configurations/all_sensors.json',
+    weather="examples/configurations/weather.json",
+    ego="examples/configurations/vehicle.json"
+)
+
+simulator.start()
+```
+
+Here the simulator is being created from several of the provided configuration 
+files for "Closed Loop" control of the EGO vehicle. 
+
+### Subscribing to Sensors
+
+Once the client is connected to the `simulator` object, sensors can be 
+configured and a callback function connected to their output:
```

Comment:
Once the client is connected to the `simulator` object, sensors can be 
configured and a callback function **is** connected to their output:

Reply:
Fixed